parser/ast/ddl.go

```diff
@@ -4542,9 +4550,10 @@ func CheckRunawayAppend(ops []*ResourceGroupRunawayOption, newOp *ResourceGroupR
 type AlterResourceGroupStmt struct {
 	ddlNode
 
-	ResourceGroupName       model.CIStr
-	IfExists                bool
-	ResourceGroupOptionList []*ResourceGroupOption
+	ResourceGroupName              model.CIStr
+	IfExists                       bool
+	ResourceGroupOptionList        []*ResourceGroupOption
+	ResourceGroupRunawayOptionList []*ResourceGroupRunawayOption
```

Comment:
Why need this field? I see there are already ResourceGroupRunawayOptionList in ResourceGroupOptionList

Reply:
forget to remove it